routes/exerciseRoutes.js

```diff
@@ -77,10 +77,7 @@ router.put("/:section_id", async (req, res) => {
       },
       function (err, docs) {
         if (err) {
-
-          res.send(err);
-        } else {
-
+          res.status(422).send(err);
```

Comment:
error status 422 may make sense for a PATCH operation, but I am not sure, so maybe this should be double checked.